state_trace_nop.go

```diff
@@ -0,0 +1,51 @@
+// Copyright 2024 The etcd Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+//go:build !with_tla
+// +build !with_tla
+
+package raft
+
+import (
+	"go.etcd.io/raft/v3/raftpb"
+	"go.etcd.io/raft/v3/tracker"
+)
+
+const EnableStateTrace = false
```

Comment:
Where is `EnableStateTrace` used?

Reply:
It might be useful if application needs to know if tracing is enabled.

Reply:
Suggest to rename it to "`StateTraceDeployed`", note it's different from `StateTraceEnabled`, which is controlled by the `TraceLogger != nil`.

Reply:
Fixed.